src/pages/workspaces/workspace/analysis/modals/AzureComputeModal.js

```diff
@@ -204,9 +185,9 @@ export const AzureComputeModalBase = ({
       [viewMode === 'deleteEnvironment',
         () => Ajax().Runtimes.runtimeV2(workspaceId, currentRuntimeDetails.runtimeName).delete()], //delete runtime
       [Utils.DEFAULT, () => {
+        // TODO: We DO currently support re-attaching azure disks
```

Comment:
Do we have any ticket number we can add to this TODO and the next TODO on 201?

Reply:
NIT: If there's nothing to do can we remove the TODO part? :)

Reply:
well I think this is what [Jake's ticket](https://broadworkbench.atlassian.net/browse/IA-4052) addresses, so I added that ticket # here